src/module/Ui/templates/layout/navigation/front-page.phtml

```diff
@@ -1,6 +1,24 @@
 <?php
 /**
- * @authors Kanstantsin A Kamkou (2ka.by); Jeroen Herczeg; Timo Klement
```

Comment:
this might be code that we copied, since we don't know the author. Should we remove that notice anyway? Or do we have to keep that (because of some licensing)?

Reply:
hm... author note w/o license or something like that doesn't help us anyways... I assumed that this is some1 that contributed in the early days, but might be wrong :/ Would remove the notice

Reply:
fine by me